sources/imdsclient/src/lib.rs

```diff
@@ -1,19 +1,14 @@
 /*!
```

Comment:
It looks like some whitespace may have been accidentally removed in this docstring comment.